test/wallet-account.test.js

```diff
@@ -154,7 +154,7 @@ function setupWalletConnectionForSigning({ allKeys, accountAccessKeys }) {
             return {
                 transaction_outcome: { outcome: { logs: [] } },
                 receipts_outcome: []
-            }
+            };
```

Comment:
Thanks for keeping the refactor and the new test+functionality in separate commits! I'm curious how you feel about rebasing. My inclination would have been to rebase these lint fixes into the previous commits, rather than add a third commit. Do you have strong feelings for or against this sort of rebasing?

Reply:
>  Do you have strong feelings for or against this sort of rebasing?

I don't think there is much reason to care, I usually rebase in such cases:
1) I'm actually rebasing on upstream branch that did change.
2) commits are total mess because of experimentation, i.e. history is full of stuff like "TMP:...", "WIP: ", "try this thing", etc

rebasing lint fixes is usually PITA as they often apply to multiple previous commits